anticipy/forecast_plot.py

```diff
@@ -322,9 +322,8 @@ def _plotly_forecast_create(df_fcast, subplots, sources, nrows, ncols,
     return fig
 
 
-def plot_forecast(df_fcast, path, output='html', width=None,
-                  height=None, title=None, dpi=70, show_legend=True,
-                  auto_open=False):
+def plot_forecast(df_fcast, output, path, width=None, height=None,
```

Comment:
Shouldn't we have path=None here?

Reply:
I thought a bit about this yesterday, but I'm not sure that have path=None is the right approach, as when using None that usually means that it is not needed in most cases or it defaults to some value within the code if not provided. This is not the case in plot_forecast as the value of "path" is related to the value of "output", so it's up to the user to define exactly what he wants. That's why I also changed the order as it makes more sense to me.

Reply:
I should have explained the issue better. I raised it when I wanted to plot in a jupyter notebook and I got an error for missing parameters. The fact that I had to add `plot=None` was counter-intuitive.

So the idea was to make `path` optional, which means adding `path=None` in the function signature, as @capelastegui suggests.

Reply:
hm ok, I see. I'll proceed with the change then.